skymodel.py

```diff
@@ -0,0 +1,202 @@
+        (default is 90.0)
+
+    incl_loweratm: str
+        Flag for inclusion of molecular emission of lower atmosphere.
+        options: ['Y', 'N'] (default is 'Y')
+    incl_upperatm: str
+        Flag for inclusion of molecular emission of upper atmosphere.
+        options: ['Y', 'N'] (default is 'Y')
+    incl_airglow: str
+        Flag for inclusion of airglow continuum (residual continuum)
+        options: ['Y', 'N'] (default is 'Y')
+
+    incl_therm: str
+        Flag for inclusion of instrumental thermal radiation.
+        options: ['Y', 'N'] (default is 'N')
+        Note: This radiance component represents an instrumental effect.
+        The emission is provided relative to the other model components.
+        To obtain the correct absolute flux, an instrumental response curve
+        must be applied to the resulting model spectrum.
+        See section 6.2.4 in the SkyCalc documentation at
+        http://localhost/observing/etc/doc/skycalc/
+        The_Cerro_Paranal_Advanced_Sky_Model.pdf
+    therm_t1, therm_t2, therm_t3 : float
+        Temperature in K (default is 0.0)
+    therm_e1, therm_e2, therm_e3: float
+        In range [0,1] (default is 0.0)
+
+    vacair: str
+        In regards to the wavelength grid.
+        options: ['vac', 'air] (default is 'vac')
+    wmin: float
+        Minimum wavelength (nm) in the wavelength grid.
+        Must be in range [300.0,30000.0] and < wmax
+        (default is 300.0)
+    wmax: float
+        Maximum wavelength (nm) in the wavelength grid.
+        Must be in range [300.0,30000.0] and > wmin
+        (default is 2000.0)
+    wgrid_mode: str
+        Mode of the wavelength grid.
+        options: ['fixed_spectral_resolution','fixed_wavelength_step', 'user']
+        (default is 'fixed_wavelength_step')
+    wdelta: float
+        Wavelength sampling step dlam in range [0,30000.0] (nm/step)
+        (default is 0.1)
+    wres: int
+        lam/dlam where dlam is wavelength step.
+        In range [0,1.0e6] (default is 20000)
+    wgrid_user: list of floats
+        default is [500.0, 510.0, 520.0, 530.0, 540.0, 550.0]
+
+    lsf_type: str
+        Line spread function type for convolution.
+        options: ['none','Gaussian','Boxcar'] (default is 'none')
+    lsf_gauss_fwhm: float
+        Gaussian full-width half-max for line spread function wavelength bins.
+        Range > 0.0 (default is 5.0)
+    lsf_boxcar_fwhm: float
+        Boxcar full-width half-max for line spread function wavelength bins.
+        Range > 0.0 (default is 5.0)
+
+    observatory: str
+        Observatory where observation takes place.
+        Options are 'paranal', 'lasilla', 'armazones' (default is 'paranal')
+
+    Returns
+    -------
+    trans_waves, transmission: tuple of arrays of floats
+        'trans_waves' is an array of wavelengths in angstroms (float),
+        'transmission' is an array of fractional atmospheric
+        transmittance (float).
+
+    """
+
+    params = locals()
```

Comment:
What is this step doing? I might be misunderstanding it, but I'm not sure it's necessary.

Reply:
`locals()` makes a dictionary of all of the input parameters. Since I reference `params` multiple times in the following lines, I thought it would be better form to call `locals()` just once.